src/index.css

```
@@ -11,13 +11,29 @@ html {
   box-sizing: border-box;
 }
 
+:root {
+  --primary-color: rgba(20, 29, 47, 1);
+  --secondary-color: #1e2a47;
```

```suggestion
  --primary-color: rgb(243 244 246);
  --secondary-color: white;
```
Try to change the color and let me know what you think about it? Is it looking good?

---

Is the color you are referring to for the light theme? I think yes, the color combination is more fitting than before

---

Yes change it and update the PR.